policy/policyd_test.go

```diff
@@ -482,12 +489,13 @@ func Test_policy_UpdatePolicy(t *testing.T) {
 			return test{
 				name: "Update policy success",
```

Comment:
wrong naming?
`"Update policy success"` -> `"Update error, context timeout"`

Reply:
LGTM, @kevindiu  can you fix it?